Comic_last_five/main.go

```
@@ -0,0 +1,84 @@
+/*
+Fetch the 5 latest xkcd comics
+
+API description: https://xkcd.com/json.html
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+// sajt koji od json-a pravi go strukturu https://mholt.github.io/json-to-go/
+
+// Comic je struktura koja odgovara json-u sa linka
+type Comic struct {
+	Month      string `json:"month"`
+	Num        int    `json:"num"`
+	Link       string `json:"link"`
+	Year       string `json:"year"`
+	News       string `json:"news"`
+	SafeTitle  string `json:"safe_title"`
+	Transcript string `json:"transcript"`
+	Alt        string `json:"alt"`
+	Img        string `json:"img"`
+	Title      string `json:"title"`
+	Day        string `json:"day"`
+}
+
+func main() {
+	theFiveLatest := fetchTheFiveLatest(615)
+
+	comicOne := getContent(theFiveLatest[0])
+	comicTwo := getContent(theFiveLatest[1])
+	comicThree := getContent(theFiveLatest[2])
+	comicFour := getContent(theFiveLatest[3])
+	comicFive := getContent(theFiveLatest[4])
+
+	fmt.Println(comicOne)
+	fmt.Println(comicTwo)
+	fmt.Println(comicThree)
+	fmt.Println(comicFour)
+	fmt.Println(comicFive)
+
+}
+
+// getContent uzimam json podatke sa linka
+func getContent(jsonURL string) Comic {
+	loadData, err := http.Get(jsonURL)
+
+	if err != nil {
+		// panicim jer ako ovo ne radi nema potrebe da se pokrece app
+		// da li moze ovako ili moram da printam error?
+		panic(err.Error())
```

the function should return the error, and you should handler the error (panic or print) in `main()`